src/maps/mod.rs

```diff
@@ -249,9 +264,8 @@ impl PerfMap {
                 Ok(PerfMap {
                     name: map_name.to_owned(),
                     base_ptr: AtomicPtr::new(base_ptr),
-                    page_count,
+                    buffer_size,
                     page_size,
```

Comment:
Those are the names used in `libbpf` which this library takes inspiration from but I have renamed `mmap_size` to be `buffer_size` in the iterator since it is confusing. In the place before it was named `mmap_size` because that was how much memory was being allocated through `mmap`. In the iterator portion it was called `mmap_size` because that is how much memory there were in the buffers created by `mmap` which is one page less than the total allocated size. Good catch!